docs/concepts/scopes.md

```diff
@@ -0,0 +1,129 @@
+            - name: hostname
+              value: example.com
+            - name: path
+              value: /
+            - name: service_port
+              value: 9999
+      <b style="color: blue">applicationScopes:</b>
+        - my-health-scope
+</pre>
+
+For more on using specific scopes, refer to the sections below.
+
+## Supported scopes
+
+Here's how to get info on the scopes supported on your Rudr installation.
+
+**List supported scope types**:
+
+```console
+$ kubectl get scopes
+```
+
+**Show the schema details of a scope instance:**
+
+```console
+$ kubectl get <scope-type> <scope-name> -o yaml
+````
+
+... where `<scope-type>` is one of the Rudr-supported scopes types (per `kubectl get scopes`), and `<scope-name>` is the *name* of a particular scope configuration instance. For example: `kubectl get health my-health-scope -o yaml`.
+
+## Health scope
+
+You can use the *HealthScope* controller to periodically check the aggregate health of components within your application. For a full walkthrough, see [Health Scope Controller](../../healthscope/README.md)
+
+### Installation
+
+Use [Helm 3](https://v3.helm.sh/) to install the Rudr *HealthScope*:
+
+```cmd
+helm install healthscope ./charts/healthscope
+```
+
+### Properties
+
+| Name | Description | Allowable values | Required | Default |
+| :-- | :--| :-- | :-- | :-- |
+| **probe-method** | The method to probe the components, e.g. 'httpGet'. | string | &#9745; | |
+| **probe-endpoint** | The endpoint to probe from the components, e.g. '/v1/health'. | string | &#9745; | |
+| **probe-timeout** | The amount of time in seconds to wait when receiving a response before marked failure. | int | | |
+| **probe-interval** | The amount of time in seconds between probing tries. | int |||
+| **failure-rate-threshold** | If the rate of failure of total probe results is above this threshold, declared 'failed'. | double |||
+| **healthy-rate-threshold** | If the rate of healthy of total probe results is above this threshold, declared 'healthy'. | double |||
+| **healthThresholdPercentage** | The % of healthy components required to upgrade scope. | double |||
+| **requiredHealthyComponents** | Comma-separated list of names of the components required to be healthy for the scope to be health. | string |||
+
+[Here's an example](../../examples/health-scope-config.yaml) of a health scope configuration. Once installed (`kubectl apply -f <health-scope-config>.yaml`) you would attach this to a component within the application configuration, similar to this [example](../../examples/first-app-config.yaml):
+
+```yaml
+# Example component scope assignment
+apiVersion: core.oam.dev/v1alpha1
+kind: ApplicationConfiguration
+metadata:
+  name: first-app
+spec:
+  components:
+    - name: helloworld-python-v1
+      instanceName: first-app-helloworld-python-v1
+      parameterValues:
+        # ...
+      traits:
+        # ...  
+      applicationScopes:
+        - my-health-scope
+```
```

Comment:
check the result of health instance created here

Reply:
Good call!